training/setup/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "wml": {
```

Comment:
Should these be part of this? Seems like credential leaking here @splovyt @ptitzler ?

Reply:
@ptitzler how did this occur?